criu/fdstore.c

```diff
@@ -49,8 +53,16 @@ int fdstore_init(void)
 		return -1;
 	}
 
-	if (setsockopt(sk, SOL_SOCKET, SO_SNDBUFFORCE, &buf[0], sizeof(buf[0])) < 0 ||
-	    setsockopt(sk, SOL_SOCKET, SO_RCVBUFFORCE, &buf[1], sizeof(buf[1])) < 0) {
+	if (has_cap_net_admin(opts.cap_eff)) {
```

Comment:
nit: if (!opts.unprivileged ||  has_cap_net_admin(opts.cap_eff))

Reply:
I don't have a strong opinion about this, but since root has all caps just checking caps is simpler than checking two conditions where one implies the other. Or are you suggesting that we should only ever look at the caps data in unprivileged mode? The caps data is initialized on all paths currently.

Reply:
I mean that we should only ever look at the caps data in unprivileged mode. Otherwise, we expect a complete set of caps, and we have to fail if that isn't true.

Reply:
Done, all checks that look at caps now only do so if unprivileged. Thanks.